gradle-module-plugin/src/main/kotlin/io/ia/sdk/gradle/modl/IgnitionModlPlugin.kt

```diff
@@ -286,6 +289,12 @@ class IgnitionModlPlugin : Plugin<Project> {
 
         p.logger.info("Setting up Java tasks on ${p.path}")
 
+        p.tasks.named("jar") {
+            if (it is Jar) {
+                it.archiveFileName.set("${p.name}-${settings.moduleVersion.get()}.jar")
```

Comment:
This feels a bit heavy, should this be a convention instead?

Reply:
Ya, that's a good point.  Update incoming.